course/src/isometry.cpp

```diff
@@ -1,18 +1,110 @@
 /*
  * Isometry library
  * Copyright 2020 Ekumen Inc.
- * Author: Agustin Alba Chicar, 2019
- * Author: Gerardo Puga, 2020
+ * Author: Alexis Pojomovsky, 2020
  */
 
+#include <cmath>
+#include <iomanip>
+
 #include <isometry/isometry.hpp>
 
 namespace ekumen {
 namespace math {
 
-/*
- * your code here
- */
+Isometry::Isometry() : rotation_{Matrix3::kZero}, translation_{Vector3()} {}
```

Comment:
That was also my original intuition, but by doing so there are some failing test cases:
```
/home/alexis/cppl1_q12020/course/test/src/isometry_TEST.cpp:101: Failure
      Expected: t7.rotation()[2][2]
      Which is: 1
To be equal to: 0
/home/alexis/cppl1_q12020/course/test/src/isometry_TEST.cpp:105: Failure
      Expected: t8.rotation()[2][2]
      Which is: 1
To be equal to: 0
```

Reply:
Go with your hunch; lines 101 and 105 are wrong. Please fix the test in your repo so that your corrected code passes the test.
```
  EXPECT_EQ(t7.rotation()[2][2], 0); // <- should compare against 1.0
 ...
  EXPECT_EQ(t8.rotation()[2][2], 0); // <- should compare against 1.0
```